pkg/util/mapz/map.go

```diff
@@ -23,3 +24,11 @@ func DecodeStructToMap(structVars any) (map[string]interface{}, error) {
 	}
 	return rawConfigVars, nil
 }
+
+// MergeMaps merge two maps
+// if there are same keys in two maps, the key of second one will overwrite the first one
+func MergeMaps(m1 map[string]any, m2 map[string]any) map[string]any {
```

Comment:
When calling this function, mapz.MergeMaps(m1, m2) contains redundant information(mapz, merge maps)

mapz.Merge(m1, m2) could be simpler without losing the meaning.